src/Visitor.php

```diff
@@ -879,28 +879,26 @@ private function getCleanCommentsNode(Node $node): Node
             return $node;
         }
 
-        // Remove "//" comments.
         $comments = [];
         foreach ($node->getComments() as $comment) {
-            if (strpos(trim($comment->getText()), '//') === 0) {
+            $commentText = trim($comment->getText());
+
+            // Strip out comments that are not doc comments.
```

Comment:
```suggestion
            // Strip out comments that are not PHPDoc comments.
```
https://docs.phpdoc.org/guide/references/phpdoc/index.html